test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/__init__.py

```diff
@@ -17,4 +17,4 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
-__version__ = '0.0.1b'
+__version__ = '0.1.0'
```

Comment:
I don't think this is used anywhere, so it can be removed.

Reply:
I removed it in 74682b3e1af3db3d02d4219c8905a5144be99269.